Formula/bats-core.rb

```diff
@@ -0,0 +1,27 @@
+class BatsCore < Formula
+  desc "Bash Automated Testing System"
+  homepage "https://github.com/bats-core/bats-core"
+  url "https://github.com/bats-core/bats-core/archive/v1.0.1.tar.gz"
+  sha256 "821626f1e5058a4f25a95722399b460942f27535186a815a279e518b503f8de7"
```

Comment:
Can we add `head` to support `--HEAD` installs?

Reply:
@jasonkarns I can, but [CONTRIBUTING.md](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md#add-a-new-formula-for-foo-version-234-from-url)
says to run 
`$ brew audit --new-formula bats-core`
which then breaks with:
```
bats-core:
  * Formulae should not have a HEAD spec
Error: 1 problem in 1 formula
```

Reply:
No, Jenkins does not like it easier for new Formulae